src/Datadog.Trace.ClrProfiler.Managed/ScopeFactory.cs

```diff
@@ -60,13 +62,16 @@ public static Scope CreateOutboundHttpScope(Tracer tracer, string httpMethod, Ur
                     UriHelpers.CleanUri(requestUri, removeScheme: true, tryRemoveIds: true));
 
                 span.SetTag(Tags.SpanKind, SpanKinds.Client);
-                span.SetTag(Tags.HttpMethod, httpMethod?.ToUpperInvariant());
-                span.SetTag(Tags.HttpUrl, UriHelpers.CleanUri(requestUri, removeScheme: false, tryRemoveIds: false));
+                span.SetTag(Tags.HttpMethod, httpMethod.ToUpperInvariant());
```

Comment:
Is there still a chance for `httpMethod` to be `null` here? I think if both `httpMethod` and `parent.GetTag(Tags.HttpMethod)` are `null` then that `string.Equals()` will return `true`...
```suggestion
                span.SetTag(Tags.HttpMethod, httpMethod?.ToUpperInvariant());
```